<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPane</class>
 <widget class="QWidget" name="InputPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="LeftPaneWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="go-previous">
            <normaloff>../../PulseaudioPane</normaloff>../../PulseaudioPane</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Input</string>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Keyboard</string>
         </property>
         <property name="icon">
          <iconset theme="preferences-desktop-keyboard">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shortcuts</string>
         </property>
         <property name="icon">
          <iconset theme="preferences-desktop-shortcuts"/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mouse and Touchpad</string>
         </property>
         <property name="icon">
          <iconset theme="preferences-desktop-mouse">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gestures</string>
         </property>
         <property name="icon">
          <iconset theme="preferences-desktop-gestures">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="tStackedWidget" name="settingsStack">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="KeyboardPane" name="keyboardPane"/>
     <widget class="ShortcutPane" name="shortcutsPane"/>
     <widget class="MousePane" name="mousePane"/>
     <widget class="GesturePane" name="gesturePane"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KeyboardPane</class>
   <extends>QWidget</extends>
   <header>Input/keyboardpane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GesturePane</class>
   <extends>QWidget</extends>
   <header>Input/gesturepane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MousePane</class>
   <extends>QWidget</extends>
   <header>Input/mousepane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShortcutPane</class>
   <extends>QWidget</extends>
   <header>Input/shortcutpane.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>settingsStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
